django/core/cache/backends/redis.py

```diff
@@ -58,7 +59,7 @@ def __init__(
             parser_class = import_string(parser_class)
         parser_class = parser_class or self._lib.connection.DefaultParser
 
-        self._pool_options = {"parser_class": parser_class, "db": db}
+        self._pool_options = {"parser_class": parser_class, "db": db, **options}
```

Comment:
We need to pass options directly to the connection pool because `Redis` ignores `kwargs` when `connection_pool` is specified as in our case, see [`Redis.__init_()`](https://github.com/redis/redis-py/blob/42b937fa1f73f3e8251eba5ec8ead2fcbaec7c50/redis/client.py#L946-L1022).

Reply:
We can remove `db` from this line and from the arguments for `RedisCacheClient.__init__()` as we're not doing anything special here and it'll be handled by `**options`. We need to keep `parser_class` as we're allowing it to be provided as a dotted string for lazy import.

Reply:
Removed.

Reply:
Still seems to be there?

Reply:
Ah, yes :facepalm: That's how it ends when you forgot to commit :laughing: